tests/ui/suggestions/return-bindings.stderr

```diff
@@ -22,8 +22,8 @@ LL | |     } else {
    |
 help: consider returning the local binding `s`
    |
-LL ~     let s: String = if let Some(s) = opt_str {
-LL +         s
+LL |     let s: String = if let Some(s) = opt_str {
+LL ~         s
 LL ~
```

Comment:
This looks like there might be something iffy, related to the other multiline "issues".

Reply:
Yes, it's the same root cause.